observability/pillars/dashboard.md

```diff
@@ -0,0 +1,58 @@
+# Dashboard and Visualization
+
+## Overview
+
+TLDR - Dashboard to show patterns. What you want to achieve here is to derive a pattern - user, usage, search or anything that you have defined as your KPI.
+
+## Best Practices
+
+1. Identify your goal or KPI.
+2. Ask yourself questions that can help you reach your goal or KPI. This may sound counter intuitive, the more questions you ask the better the outcome will be.
+3. Validate your questions. Are these necessary in finding a common pattern?
+4. Build simple chart, have more of them. Rather than having a complicated all in one dashboard, seperate your dashboard into sections. Azure Workbooks is highly recommended as it supports markdown.
+5. Observe the dashboard that you built. Is the data reflecting what you are set out to answer? If not make small adjustment.
+
+Always remember this process takes time. Building dashboard is easy, building an obserable dashboard to show pattern is hard.
+
+## Recommended Tools
+
+- [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/overview) - Umbrella of services including system metrics, log analytics and more.
+- [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) - An extensible Application Performance Management (APM) service used to monitor live applications, and a feature of inside Azure Monitor.
+- [Azure Log Analytics](https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/log-query-overview) - Run advanced queries against the data collected via Azure Monitor Logs.
+- [Azure Monitor Workbooks](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/workbooks-overview) - Visualizations for your logged data, tightly interated with Azure services and higly customizable.
+
+### Azure Monitor Workbooks
```

Comment:
I don't see these sections as necessary as they are getting into tool-specific features/guidance. We should be focusing on generic guidance about how to construct dashboards and their uses, vs specific tool features. The links used in the Recommended Tools section can lead readers to the feature details of the specific tools.

Reply:
Thanks Ken. I would want to keep this in providing reader an example of tools & how it's features can help build an observable dashboard. Key here is observable. There are a lot of dashboard tools out there that does the job well. I do agree with you this is not a section to promote tools. I'll probably remove the tool features and give scenario specific examples in the next revision.